docker-compose.yml

```
@@ -12,23 +12,7 @@ services:
-  redis:
-    image: redis:latest
-    container_name: bluelight-hub-redis
-    volumes:
-      - redis_data:/data
-    restart: unless-stopped
-    ports:
-      - '${REDIS_PORT:-9036}:6379'
-    command: redis-server --appendonly yes --appendfsync everysec
-    healthcheck:
-      test: ['CMD', 'redis-cli', 'ping']
+      test: [ 'CMD-SHELL', 'pg_isready -U bluelight' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)